lib/concentrate/vehicle_position.ex

```diff
@@ -41,6 +41,11 @@ defmodule Concentrate.VehiclePosition do
     :scheduled_headway_secs,
     :sources,
     :data_discrepancies,
+    :load,
+    :route_has_reliable_crowding_data,
+    :capacity,
+    :occupancy_status,
+    :occupancy_percentage,
```

Comment:
Did you consider whether these would be cleaner as a struct/map in a single nilable field, so they could all be `nil` together instead of each field being `nil` one-by-one?

It could reduce the amount of boilerplate in tests, and could make things like this typesafe:
```ts
  // If vehicle.load isn't null, vehicle.occupancyStatus shouldn't be either.
  return vehicle.occupancyStatus! 
```

but I don't know if the data is consistent enough or if our vehicle_position processing code could handle it cleanly.

Reply:
@skyqrose yes, I think you're right, we have a data clump here.

Reply:
Although we do still have to do that slightly inelegant `!` there.